e2e/tests/core/03-connection/connection_test.go

```diff
@@ -0,0 +1,131 @@
+package connection
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	paramsproposaltypes "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	"github.com/strangelove-ventures/ibctest/v6/ibc"
+	"github.com/strangelove-ventures/ibctest/v6/test"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/ibc-go/e2e/testsuite"
+	"github.com/cosmos/ibc-go/e2e/testvalues"
+	transfertypes "github.com/cosmos/ibc-go/v6/modules/apps/transfer/types"
+	connectiontypes "github.com/cosmos/ibc-go/v6/modules/core/03-connection/types"
+	ibctesting "github.com/cosmos/ibc-go/v6/testing"
+)
+
+func TestConnectionTestSuite(t *testing.T) {
+	suite.Run(t, new(ConnectionTestSuite))
+}
+
+type ConnectionTestSuite struct {
+	testsuite.E2ETestSuite
+}
+
+// QueryMaxExpectedTimePerBlockParam queries the on-chain max expected time per block param for 03-connection
+func (s *ConnectionTestSuite) QueryMaxExpectedTimePerBlockParam(ctx context.Context, chain ibc.Chain) uint64 {
+	queryClient := s.GetChainGRCPClients(chain).ParamsQueryClient
+	res, err := queryClient.Params(ctx, &paramsproposaltypes.QueryParamsRequest{
+		Subspace: "ibc",
+		Key:      string(connectiontypes.KeyMaxExpectedTimePerBlock),
+	})
+	s.Require().NoError(err)
+
+	// removing additional strings that are used for amino
+	delay := strings.ReplaceAll(res.Param.Value, "\"", "")
+	time, err := strconv.ParseUint(delay, 10, 64)
+	s.Require().NoError(err)
+
+	return time
+}
+
+// TestMaxExpectedTimePerBlock tests changing the MaxExpectedTimePerBlock param using a governance proposal
+func (s *ConnectionTestSuite) TestMaxExpectedTimePerBlock() {
```

Comment:
We should change this method name to have a `Param` suffix to align with others in transfer tests for example

```suggestion
func (s *ConnectionTestSuite) TestMaxExpectedTimePerBlockParam() {
```